src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -1243,8 +1243,8 @@ public void Search(string searchText)
             {
                 await NuGetUIThreadHelper.JoinableTaskFactory.SwitchToMainThreadAsync();
 
-                _windowSearchHost.Activate();
-                _windowSearchHost.SearchAsync(new SearchQuery { SearchString = searchText });
+                _windowSearchHost?.Activate();
```

Comment:
Sometimes VS crashes and I didn't know why. Added just in case

Reply:
Finally got a stack trace:

```
Exception type:   System.InvalidOperationException
Message:          Cannot start a new search for 'Azure.Storage.Blobs' because there is another search in progress for 'Azure.Storage.Blobs'.
```

I need a way to stop current running search.

Then, I think I can safely remove this null check.

Reply:
Fixed by terminating the search